courses/free-courses-pl.md

```diff
@@ -114,12 +114,15 @@
 
 ### Python
 
-* [Kurs online Python dla początkujących](https://www.flynerd.pl/tag/python-kurs) - Małgorzata Łyczywek AKA Rita (HTML)
+* [Jak nauczyć się programowania](https://www.youtube.com/watch?v=eB3r2NQwNi4&t=5300s)
+* [Kurs online Python dla początkujących](https://www.flynerd.pl/tag/python-kurs)
 * [Kurs Python](https://www.youtube.com/playlist?list=PL3yDCQ6GKeEyBOF0gZyBvihDv6n0GNsdm)
 * [Kurs Python - Darmowy Po Polsku](https://www.youtube.com/playlist?list=PL_dDQ_G9rdI6dQsDkwqSQyAeXY3uUrWzp)
-* [Kurs Python 3](https://www.youtube.com/playlist?list=PLdBHMlEKo8UcOaykMssI1_X6ui0tzTNoH) - Piotr Baja
+* [Kurs Python 3](https://www.youtube.com/playlist?list=PLdBHMlEKo8UcOaykMssI1_X6ui0tzTNoH)
 * [Python 3 - Kurs wideo](https://www.youtube.com/playlist?list=PLbOPmSDkHx2pCboufcEKkinpUuramshmr)
-* [Raspberry Pi kurs od podstaw](https://forbot.pl/blog/kurs-raspberry-pi-od-podstaw-wstep-spis-tresci-id23139) - Piotr Bugalski (FORBOT.pl)
+* [Raspberry Pi kurs od podstaw](https://forbot.pl/blog/kurs-raspberry-pi-od-podstaw-wstep-spis-tresci-id23139)
```

Comment:
why did you remove author attribution?

Reply:
I removed it out while trying to fix the linter error and realized it was already there before my pull request. I made some adjustments to investigate the issue that's preventing deployment.